recipes/mdsrv/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "MDsrv" %}
+{% set version = "0.3.5" %}
+{% set sha256 = "3dd218ec0d62cf47061699b8ab33715d6875fad824091f2ab37e546467fef0c3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install 
```

Comment:
Please use:

```
script: python setup.py install --single-version-externally-managed --record record.txt
```

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L26

since it appears that this package uses `setuptools`.